frontend-vue/src/router.js

```diff
@@ -1,23 +1,37 @@
 import Vue from 'vue'
 import Router from 'vue-router'
-import Home from './views/Home.vue'
+import Home_page from './views/Home_page.vue'
+import Char_input from './views/Char_input.vue'
+import Output_page from './vews/Output_page.vue'
+
 
 Vue.use(Router)
 
 export default new Router({
   routes: [
-    {
-      path: '/',
-      name: 'home',
-      component: Home
-    },
     {
       path: '/about',
       name: 'about',
       // route level code-splitting
       // this generates a separate chunk (about.[hash].js) for this route
       // which is lazy-loaded when the route is visited.
       component: () => import(/* webpackChunkName: "about" */ './views/About.vue')
+    },
+    {
+      path:'/',
+      name:'Home_page',
+      component: Home_page
+    },
+    {
+      path:'/Char_input',
```

Comment:
URL paths are typically lower case (unless if you're accessing that is specifically named in uppercase, like the URL for this repo), but up to you if you want to change it.